js/libs/core.js

```diff
@@ -332,6 +332,16 @@ Fliplet.FormBuilder = (function() {
           return;
         }
 
+        if (['flDate', 'flTime'].includes(this._componentName) && this.isApplySpecificDateField && !this.value) {
```

Comment:
`includes` is not supported in IE11

Reply:
> `includes` is not supported in IE11

do we support IE11?

Reply:
@sofiiakvasnevska please confirm @skrupnyk has gone through the onboarding material.

Reply:
fixed